src/Serialization/BigNum.purs

```diff
@@ -0,0 +1,15 @@
+module Serialization.BigNum
+  ( convertBigNum
+  ) where
+
+import Prelude
+import Serialization.Types (BigNum)
+import FfiHelpers
+import Data.BigInt as BigInt
+import Data.BigInt (BigInt)
+import Data.Maybe
+
+convertBigNum :: BigInt -> Maybe BigNum
```

Comment:
Most (if not all?) of the naming conventions for `convertSomething`  have so far been the "same" thing from foreign data types to/from Purescript types. These two are however different types, is `bigNumFromBigInt` or `bigIntToBigNum` perhaps a bit more descriptive? They are a little long winded however.